<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Details</class>
 <widget class="QDialog" name="Details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>110</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsCountryTextLable">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsCityTextLable">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>110</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line_20">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTime1Lable">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6:00 - 8:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTemp1Lable">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>14 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetails1ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mWind1InfoLable">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wind - 2 m/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mPreassure1InfoLable">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>792 N/m2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mHumidity1InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Humidity 45%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="Line" name="line_21">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTime2Lable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>8:00 - 10:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTemp2Lable">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>17 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetails2ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mWind2InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wind - 2 m/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mPreassure2InfoLable">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>792 N/m2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mHumidity2InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Humidity 46%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="Line" name="line_22">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTime3Lable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10:00 - 12:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTemp3Lable">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>21 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetails3ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mWind3InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wind - 2 m/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mPreassure3InfoLable">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>792 N/m2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mHumidity3InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Humidity 42%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="Line" name="line_23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTime4Lable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>12:00 - 14:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTemp4Lable">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>24 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetails4ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mWind4InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wind - 2 m/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mPreassure4InfoLable">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>792 N/m2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mHumidity4InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Humidity 53%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_17">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="Line" name="line_24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTime5Lable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>14:00 - 16:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTemp5Lable">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>22 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetails5ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mWind5InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wind - 2 m/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mPreassure5InfoLable">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>792 N/m2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mHumidity5InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Humidity 54%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_18">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="Line" name="line_25">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTime6Lable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>16:00 - 18:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetailsTemp6Lable">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>21 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDetails6ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mWind6InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wind - 2 m/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mPreassure6InfoLable">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>792 N/m2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mHumidity6InfoLable">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Humidity 50%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
